index.html

```diff
@@ -223,6 +223,7 @@ <h1>Download</h1>
 					<button class="nav-link filter" data-filter=".allo">Allo</button>
 					<button class="nav-link filter" data-filter=".asus">ASUS</button>
 					<button class="nav-link filter" data-filter=".nanopi">NanoPi</button>
+					<button class="nav-link filter" data-filter=".radxa">Radxa</button>
```

Comment:
Ah this doubles with four lines above. The text there is "ROCK Pi" and indeed it makes sense to rename it to "Radxa".